CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## [Unreleased]
 
-- No changes yet.
+- Fix archive and git inputs so that `--path` and `--exclude-path` paths are relative to
+  the `#subdir` rather than the root of the input. This fixes a unintended behavior change that was introduced in `v1.32.0`.
```

Comment:
`a` -> `an` (and wrap)
```suggestion
  the `#subdir` rather than the root of the input. This fixes an unintended behavior change
  that was introduced in `v1.32.0`.
```